### DISCUSSION TITLE
Approaches RNAV/LNAV

### DISCUSSION THREAD

**Discussion by PAGIS30** [flybywiresim/aircraft#7266] [Help]
Hello
For RNAV/LNAV approaches I encounter quite a few difficulties in the final phase of the descent.
I think I correctly fill in the MCDU criteria but either the approach to the runway is too high or, more often than not, too low with therefore a missed landing.
Should I blame the driver :) or the a32nx which is not yet quite calibrated on this point?
What about you?
Thank you in advance for your opinions

**Comment by ExampleWasTaken** [comment]
The A32NX doesn not yet support RNP approaches as VNAV is still experiemental and not complete.
